lib/kafka/broker_pool.rb

```diff
@@ -2,12 +2,14 @@
 
 module Kafka
   class BrokerPool
-    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:)
+    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:, ssl: nil, ssl_context: nil)
```

Comment:
because you don't actually always need an ssl context. If your broker doesn't use ssl client certs for auth, but instead just wants encrypted messages, then you don't need a context.